engine/src/flutter/shell/platform/embedder/embedder_semantics_update.cc

```diff
@@ -126,6 +126,44 @@ FlutterSemanticsFlag SemanticsFlagsToInt(const SemanticsFlags& flags) {
   return static_cast<FlutterSemanticsFlag>(result);
 }
 
+const FlutterSemanticsFlags convertToFlutterSemanticsFlags(
```

Comment:
Since this function isn't used outside this translation unit (file), it should be `static` -- or, in more modern C++ in an anonymous namespace, usually at the top of the file. So:
```cc
namespace {
FlutterSemanticsFlags FlutterSemanticsFlagsFromSemanticsFlags(const SemanticsFlags& source) {
  // ...
}
}  // namespace